packages/boot/tools/supports.ts

```diff
@@ -587,6 +587,7 @@ export const makeSwingsetTestKit = async (
 
   console.timeLog('makeBaseSwingsetTestKit', 'buildSwingset');
 
+  await controller.run();
```

Comment:
Please leave a comment explaining why this is necessary, including why (if so?) it needs to run before `makeBootstrapRunUtils`.

Reply:
This was extracted _from_ makeBootstrapRunUtils, which before this PR included `mutex.put(controller.run().catch(logRunFailure))` but is now changing to `mutex.put('dummy result')`. I'll check if `makeSwingsetTestKit` works without it, but if not then what kind of a comment do you suggest?

Reply:
It wasn't clear from my comment: I don't understand why this line would be necessary. So I was requesting an explanation for why it's included.

I've tested this branch locally without it and it passes so now my request is to remove it :) 
```suggestion
  await controller.run();
```

fwiw, I don't see `controller.run` in the boot package in master.

Reply:
Excellent, my own run just finished and confirmed the same.

> fwiw, I don't see `controller.run` in the boot package in master.

That makes sense; all instances (sometimes as `c.run()`) are probably inside `makeRunUtils` or elsewhere in packages/SwingSet/.

Reply:
Hmm, but the one in packages/boot/test/upgrading/upgrade-vats.test.ts **is** necessary, which makes me think that `makeSwingsetTestKit` just isn't being exercised in the right way. I'm therefore inclined to keep this initial `controller.run()`, and asking you for an explanatory comment if you still consider that to be necessary.

Reply:
> just isn't being exercised in the right way

Valid but unconfirmed hypothesis.

> an explanatory comment if you still consider that to be necessary

I do consider it necessary because I'm sure I would try removing it, see that all the tests are passing, send it for PR and it would probably be approved.

So something like,
```
// XXX we're not certain this is necessary. Tests pass without it but we suspect that
// `makeSwingsetTestKit` just isn't being exercised in the right way
```